card/lib/card/loader.rb

```diff
@@ -15,12 +15,53 @@ def paths
   end
 
   module Loader
-
     class << self
       def load_mods
         load_set_patterns
         load_formats
         load_sets
+
+        if ENV['RAILS_ENV'] == 'development'
+          update_machine_output_hack
+        end
+      end
+
+      def update_machine_output_hack
+        update_script_output
+        update_style_output
+      end
+
+      def update_script_output
+        script = Card['*all+*script']
+        return unless (mtime_output = script.machine_output_card.updated_at)
+        ['wagn_mod.js.coffee', 'wagn.js.coffee',
+         'script_card_menu.js.coffee'].each do |name|
+          mtime_file = File.mtime(
+            "#{Cardio.gem_root}/mod/03_machines/lib/javascript/#{name}"
+          )
+          if mtime_file > mtime_output
+            script.update_machine_output
+            break
+          end
+        end
+      end
+
+      def update_style_output
```

Comment:
Assignment Branch Condition size for update_style_output is too high. [15.39/15]